src/openrct2/peep/Staff.cpp

```diff
@@ -84,16 +84,22 @@ static inline void staff_autoposition_new_staff_member(rct_peep * newPeep)
 
     newPeep->state = PEEP_STATE_FALLING;
 
-    sint16     x, y, z;
-    uint32     count = 0;
-    uint16     sprite_index;
-    rct_peep * guest = nullptr;
+    sint16				x, y, z;
```

Comment:
Your latest commit has changed the alignment here. Note that we use four spaces for indentation, not tabs.